ledger/error/src/main/scala/com/daml/error/BaseError.scala

```diff
@@ -91,9 +91,14 @@ trait LocationMixin {
 
 object BaseError {
   private val ignoreFields = Set("cause", "throwable", "loggingContext")
-  val SecuritySensitiveMessageOnApi =
+  val SanitizedSecuritySensitiveMessageOnApiPrefix =
     "An error occurred. Please contact the operator and inquire about the request"
 
+  def isSanitizedSecuritySensitiveMessage(msg: String): Boolean = {
+    // TODO error codes: Check that suffix is a correlation id and nothing else.
```

Comment:
If you want to do it, please do it in this ticket. If you are unsure, I would ask offline.

Reply:
This is a TODO to improve this function to check the full `msg` string, not just its prefix.